include/mf_elasticity.h

```diff
@@ -143,6 +143,18 @@ namespace FSI
     using VectorType          = LinearAlgebra::distributed::Vector<Number>;
     using VectorizedArrayType = VectorizedArray<Number>;
     using LevelVectorizedArrayType = VectorizedArray<LevelNumber>;
+    using FECellIntegrator         = FEEvaluation<dim,
```

Comment:
From which code do you have this name? I am just curious ;)

Reply:
First seen in the Meltpool code or ExaDG  and now also in step-75. I'm not completely sure, but I guess the author/editor was always the same ;), good idea

Reply:
If you want go one step further, define it globally like this:

https://github.com/MeltPoolDG/MeltPoolDG/blob/0898378fdf1269bd9dde4eaeab9dc11a0068e7fc/include/meltpooldg/utilities/fe_integrator.hpp#L18

and drop two template parameters^^

Reply:
Ah very nice, I haven't seen that. Will definitely check it out, I haven't seen aliases using templates yet. Thanks!

Reply:
> If you want go one step further, define it globally like this:

AFAIU I cannot instantiate multiple instances of my operators in one build when defining the template parameter globally. Of course, there are now various options, but I will go with the class definition for now.

Reply:
I don't get it!? Globally makes only the non-templated version (-1, 0).

Reply:
How do you handle the case with custom templete parameters (degree, nqpoints). Say you want one version with (degree, degree+3) and maybe another one which is not compiled aka. in the hot path?

Reply:
That is a good question. The max degree is specified by this line:

https://github.com/dealii/dealii/blob/c9af205db89377b02d4f319a6e54fa9640aabeee/include/deal.II/matrix_free/evaluation_template_factory.templates.h#L29

and the degree/n_q_points combination by:

https://github.com/dealii/dealii/blob/c9af205db89377b02d4f319a6e54fa9640aabeee/include/deal.II/matrix_free/evaluation_template_factory.templates.h#L44-L52

To increase the max degree, you can proceed like this https://github.com/exadg/exadg/blob/master/include/exadg/matrix_free/resolve_templates.h and https://github.com/exadg/exadg/blob/master/include/exadg/matrix_free/resolve_templates_double_2d.cpp in your application. 

To introduce additional combinations is not possible at the moment :( We added those which we needed :)

Reply:
Ah this is very interesting. Does it recompile the whole structs in case the degree is not modified from the libraries default? May I assume yes? 

> To introduce additional combinations is not possible at the moment :( We added those which we needed :)

Honestly speaking, I think the default combinations are just fine, everything else can be handled by explicit instantiations within the code :)

Reply:
>  May I assume yes?

Yup.